src/avt_vimba_camera.cpp

```diff
@@ -666,6 +673,11 @@ int AvtVimbaCamera::getTriggerModeInt(std::string mode_str) {
   return mode;
 }
 
+void AvtVimbaCamera::queryCameraState()
```

Comment:
Since it is only query the timer tick frequency, Is this function needed?

Reply:
I had other plans to add more details here, but that can come in a later PR, I'll simplify this for now.